content/en/docs/setup/best-practices/certificates.md

```diff
@@ -64,18 +64,21 @@ If you don't wish to copy the CA private keys to your cluster, you can generate
 
 Required certificates:
 
-| Default CN                    | Parent CA                 | O (in Subject) | kind                                   | hosts (SAN)                                 |
+| Default CN                    | Parent CA                 | O (in Subject) | kind                                   | hosts (subjectAltName)                      |
 |-------------------------------|---------------------------|----------------|----------------------------------------|---------------------------------------------|
-| kube-etcd                     | etcd-ca                   |                | server, client                         | `localhost`, `127.0.0.1`                        |
+| kube-etcd                     | etcd-ca                   |                | server, client                         | `localhost`, `127.0.0.1`, (See annotation [1])           |
```

Comment:
nit: 'annotation' is a bit overloaded in Kubernetes land. Maybe footnote? Or continue to use [1] if our markdown processor supports anchors?

https://stackoverflow.com/questions/5319754/cross-reference-named-anchor-in-markdown